sklearn/inspection/_plot/tests/test_plot_partial_dependence.py

```diff
@@ -34,13 +35,19 @@ def clf_diabetes(diabetes):
     return clf
 
 
+def test_plot_partial_dependence_deprecation(pyplot, clf_diabetes, diabetes):
+    """Check that plot_partial_dependence is deprecated"""
+    with pytest.warns(FutureWarning):
+        plot_partial_dependence(clf_diabetes, diabetes.data, [0])
+
+
 @pytest.mark.filterwarnings("ignore:A Bunch will be returned")
 @pytest.mark.parametrize("grid_resolution", [10, 20])
 def test_plot_partial_dependence(grid_resolution, pyplot, clf_diabetes, diabetes):
```

Comment:
if possible, I would consider to rename the name of the test functions where `plot_partial_dependence` occur and replace by `partial_dependence_display`